markdown/events/2022/bytesize-36-multiqc.md

```diff
@@ -24,3 +24,119 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[12:04](https://www.youtube.com/watch?v=a3_IYSMrKAk&t=724)
+One of the most difficult things that MultiQC has to do is work out the name of each sample. There's no idealized situation where we just magically know what your sample identifiers are. We have to do our best guess. Usually that's by looking at either the file name of the log or trying to find the input file name and basing it on that. But of course, if you have .fastq or .bam or whatever, you have all these different extensions then they look like different identifiers. MultiQC tries to get rid of those standardized extensions so that you end up with that core identifier and then everything lines up nicely across the different modules, especially in that top table called general statistics. But it's generalized so we have to do our best and sometimes different pipelines have different extensions which are added on. If you see that happening, especially in general stats that rows aren't lining up or you see duplicate samples, which should be just one, you can tell MultiQC what your custom extensions are in this config and clean them up. You get really nice clean, short sample identifiers with no additional cruft.
+
+[13:15](https://www.youtube.com/watch?v=a3_IYSMrKAk&t=795)
+Some people get really annoyed. MultiQC has to deal with massive numbers of samples - everything say from one or two samples up to thousands - and tables get really unhelpful when they're super, super long. You can no longer summarize and take an overview view, which is the whole point of MultiQC. By default MultiQC, when a table gets to, I think 500 rows is the default, something like that, it will, instead of doing a table, generate what's called this bswan plot which is like a dot plot. If you find that really annoying, you can push up that threshold at which that switch happens to effectively disable bswan plots. A few people have done that within nf-core pipelines.
```

Comment:
```suggestion
Some people get really annoyed. MultiQC has to deal with massive numbers of samples - everything say from one or two samples up to thousands - and tables get really unhelpful when they're super, super long. You can no longer summarize and take an overview view, which is the whole point of MultiQC. By default MultiQC, when a table gets to, I think 500 rows is the default, something like that, it will, instead of doing a table, generate what's called this bswan plot which is like a dot plot. If you find that really annoying, you can push up that threshold at which that switch happens to effectively disable beeswarm plots. A few people have done that within nf-core pipelines.
```